LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2022 TypeFox GmbH
+Copyright 2023 TypeFox GmbH
```

Comment:
If you want to update the project copyright, it should be a range: 2022-2023 (but you can also leave the initial year).

The same applies to individual source files.